modules/tinymce/src/core/test/ts/browser/content/insert/InsertContentTest.ts

```diff
@@ -604,7 +604,8 @@ describe('browser.tinymce.core.content.insert.InsertContentTest', () => {
             '</td>' +
           '</tr>' +
         '</tbody>' +
-      '</table>'
+      '</table>' +
+      `<p>${String.fromCharCode(160)}</p>` // UTF-16 char code '160' is NO-BREAK SPACE (U+00A0)
```

Comment:
There is a Unicode module in Katamari that has a constant for this. Then you don't need the comment since the code would be explaining it. I wonder why the editor is configured as `raw` entities here. Ideally it would be better to be able to use entities but maybe there is a reason for it being configured that way.